### ISSUE TITLE
Can MLDB do GBM?

### ISSUE THREAD

**Issue by szilard** [szilard/GBM-perf#1]
@nicolaskruchten @jeremybarnes

Can MLDB do gradient boosting machines (GBM) similar to xgboost (with shrinkage/learning rate)?

I was trying this:
```
        "configuration": {
            "type": "boosting",
            "validation_split": 0,
            "min_iter": 100,
            "max_iter": 100,
            "weak_learner": {
                "type": "decision_tree",
                "max_depth": 10,
                "random_feature_propn": 1
            }
        },
```
(complete code [here](https://github.com/szilard/GBM-perf/blob/master/run/4-mldb.py))

I cannot find if MLDB `boosting` has a `learning_rate` parameter?

Also the above code is running very slowly, about 100sec vs 20sec for the random forest previously benchmarked [here](https://github.com/szilard/benchm-ml/issues/25). `htop` or `mpstat` shows about 15% CPU utilization.
